definitions/variable/Electricitytags_add.yaml

```diff
@@ -0,0 +1,41 @@
+
```

Comment:
Tag-files have to have a name starting with `tag_*.yaml`, in this case for example `tag_electricity-source.yaml`.

I also suggest to remove the empty line at the start of this file.
```suggestion
```